.github/workflows/build.yml

```diff
@@ -0,0 +1,103 @@
+name: Build
+
+on:
+    push:
+        branches: [ master, feature/ga ]
+    pull_request:
+    release:
+        types: [ created ]
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        name: Build and test
+        strategy:
+            matrix:
+                php: [ "7.3" ]
+                publish-phar: [ false ]
+                phar-readonly: [ false ]
+                composer-flags: [ "" ]
+                include:
+                    -   php: "7.2"
+                        composer-flags: "--prefer-lowest"
+                    -   php: "7.3"
+                        phar-readonly: true
+                    -   php: "7.4"
+                        coverage: "pcov"
+                        publish-phar: true
+                        git-fetch-depth: 0 # box/composer needs history to determine a recent git version
+                    -   php: "7.4"
+                        composer-version: "2"
+
+        steps:
+            -   uses: actions/checkout@v2
+                with:
+                    fetch-depth: "${{ matrix.git-fetch-depth:-1 }}"
```

Comment:
Github chokes on this syntax, which looks like bash syntax. This isn't bash though.